platforms/cdh6.3.0/docs/CDH-installation.md

```diff
@@ -171,7 +171,15 @@ use your data disks.
     </tbody>
    </table>
 
-3. Click the `Save Changes` button.
+3. Set the value of `HDFS Client Advanced Configuration Snippet (Safety Valve) for hdfs-site.xml`:
+   ````
+   <property><name>dfs.client.block.write.retries</name><value>6</value></property>
+   <property><name>dfs.client.block.write.locateFollowingBlock.retries</name><value>10</value></property>
+   <property><name>dfs.client.block.write.replace-datanode-on-failure.policy</name><value>ALWAYS</value></property>
+   <property><name>dfs.namenode.replication.min</name><value>2</value></property>
```

Comment:
This last property has to go into the hdfs-site.xml for the service section. As discussed in https://splicemachine.atlassian.net/browse/DB-10125 it seems that the remaining properties have to go in hbase-site.xml as they were already.

Reply:
I moved this property "dfs.namenode.replication.min" to the part "2. Change the values of these settings:"
        <tr>
            <td><code>Minimal Block Replication</code></td>
            <td><code>2</code></td>
        </tr>
is it right place? Checked in Cloudera 6.3.0 HDFS Configuration for the name.

Reply:
dfs.namenode.replication.min go into 
Configure HDFS
2. Change the values of these settings:
        <tr>
            <td><code>Minimal Block Replication</code></td>
            <td><code>2</code></td>
        </tr>

dfs.client.* are moved back into hbase-site.xml

Reply:
@ipraznik-splice it appears HBase always overrides `dfs.namenode.replication.min`, so if we want it to apply it needs to be set in hbase-site.xml.
But we should hold off on that until @OlegMazurov completes some latency tests to check whether this change has a big impact in performance.

Reply:
Oleg has completed performance tests which did not show a significant performance degradation.